server/src/test/java/org/spine3/server/storage/RecordStorageShould.java

```diff
@@ -407,5 +484,13 @@ public Timestamp getNow() {
         public Project getCounterState() {
             return getState();
         }
+
+        public int getInjectableState() {
+            return injectableState;
+        }
+
+        private void injectInjectableState(int state) {
```

Comment:
Let's keep this field. We should consider both `Message` and proto-primitive fields in the queries. Using this `Entity`, we test both via the `version` and this field.